giranezafiacre/My-Brand-Giraneza-Fiacre
# chore to finish toDo in nodejs Api

#### What does this PR do?
-it integrates retrieve data in list

#### Description of Task to be completed?
-to retrieve data of list using nodejs api

#### How should this be manually tested?
-to download codes and run 'npm install' and 'npm start' and go to postman to see data

#### Any background context you want to provide?
-N.A
#### What are the relevant pivotal tracker stories?
-N/A
#### Screenshots (if appropriate)
-N/A

#### Questions:
-N/A
